xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -1109,6 +1109,12 @@ void CLinuxRendererGL::RenderToFBO(int index, int field, bool weave /*= false*/)
 
   if (!m_fbo.fbo.IsValid())
   {
+    if (!g_Windowing.IsExtSupported("GL_EXT_framebuffer_object"))
```

Comment:
just leave the fbo untouched if not supported. trying to initialize on platforms that don't support it makes no sense